### PR TITLE
Feat/web listing page

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/product-auto-config`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5371
| License          | BSD 3-Clause

## Description

Add listing page in Web universe

⚠️ State and architecture is messy, it can be handled in another PR

### CODE REVIEW

File: packages/manager/modules/sharepoint/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@ovh-ux/web-universe-components": "^7.0.0",
     "@uirouter/angularjs": "^1.0.15",
     "angular": "^1.7.5",
+    "angular-translate": "^2.18.2",
```
Comment (antleblanc) [inline packages/manager/modules/sharepoint/package.json:26]: No update regarding `yarn.lock` file?

File: packages/manager/modules/sharepoint/src/sharepoints/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "sharepoint_title": "Sharepoint"
```
Comment (antleblanc) [inline packages/manager/modules/sharepoint/src/sharepoints/translations/Messages_fr_FR.json:2]:
```suggestion
  "sharepoint_title": "SharePoint"
```

File: packages/manager/modules/sharepoint/package.json (hunk 1 above)
Comment (marie-j) [reply packages/manager/modules/sharepoint/package.json:26]: @antleblanc It shouldn't be if it's a peer dep, right ?

File: packages/manager/apps/web/client/app/app.js [hunk 3]
```diff
@@ -529,6 +531,7 @@ angular
   .run(
     /* @ngInject */ ($state) => {
       $state.defaultErrorHandler((error) => {
+        console.log(error);
```
[the remark this answers is not in the document]
Comment (marie-j) [reply packages/manager/apps/web/client/app/app.js:534]: It's my signature though ! (I'll remove it )

**Comment by marie-j** [comment]
Rebased and fixed some conflicts

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14224.0 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14224.1 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report test-manager_2#14224.2 ✘  
* Stage 1
  * e2e -   
  * lint ✔ 
  * SemVer ✘

**Comment by ovh-cds** [comment]
CDS Report build#14276.0 ✘  
* 
  * Build ✘

**Pull request merged**